mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookWidget.mapping

```diff
@@ -32,6 +32,7 @@ CLASS net/minecraft/class_507 net/minecraft/client/gui/screen/recipebook/RecipeB
 	METHOD method_2581 drawGhostSlots (IIZF)V
 		ARG 1 left
 		ARG 2 top
+		ARG 3 isBig
```

Comment:
Sounds more like if it's in block screen or player inventory screen.

Reply:
this is same as the bigger ghost slot comment made above.